tests/unit/transform/date-test.js

```diff
@@ -17,6 +17,8 @@ test("#serialize", function(assert) {
   assert.equal(transform.serialize(undefined), null);
```

Comment:
Hmm, I would have expected that this assertion fails with the change in this PR (it doesn't because `assert.equal` uses non strict comparison).

I am wondering if `assert.deepEqual` is better suited here?